boards/arm/mimxrt1050_evk/mimxrt1050_evk_qspi.dts

```diff
@@ -8,15 +8,14 @@
 
 /delete-node/ &s26ks512s0;
 
-/ {
-	chosen {
-		/delete-property/ zephyr,flash-controller;
-		/delete-property/ zephyr,flash;
-		/delete-property/ zephyr,code-partition;
-	};
+&chosen {
```

Comment:
There is nothing wrong with `/ { chosen { ... }; };`, is there? The path to the chosen node is well known and IMO we should just refer to it by path in overlays.

Reply:
You are absoulutely right @mbolivar-nordic . Thank you for your comment. It's always good to learn.